### ISSUE TITLE
Deprecated macros page may be removed

### ISSUE THREAD

**Issue by bsmth** [mdn/content#22124]
Since https://github.com/mdn/content/pull/22104, the deprecated macros page has no content. It might be a good idea to remove this entirely.

**Comment by teoli2003** [comment]
I'm all for it. This is a remnant of the wiki era when macros were staying for a very long time, now we delete them fairly quickly, and we use GitHub (discussions/issues/…) to inform translators about the work that has to be done to replace them.

**Comment by bsmth** [comment]
Thanks @teoli2003, I've assigned myself on this one, but I can hold off on changes to see if there are any objections.

**Comment by Josh-Cena** [comment]
See https://github.com/mdn/content/pull/20771

cc @schalkneethling

**Comment by hamishwillee** [comment]
Yeah, it's not empty yet: https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Page_structures/Macros/Deprecated_macros. 

FWIW I agree we're in a pretty good state that we could delete this - we run removals as projects now, so a call for help like this is more noise than value.

**Comment by bsmth** [comment]
> Yeah, it's not empty yet

It's empty now since the latest deploy,
